utils/recog_wav.sh

```diff
@@ -54,26 +71,39 @@ decode_cmd=
 wav=$1
 download_dir=${decode_dir}/download
 
-if [ $# -gt 1 ]; then
-    echo $help_message
+if [ $# -lt 1 ]; then
+    echo "${help_message}"
     exit 1;
 fi
 
 set -e
 set -u
 set -o pipefail
 
+# Check model name or model file is set
+if [ -z $models ]; then
+    if [[ -z $cmvn || -z $lang_model || -z $recog_model || -z $decode_config ]]; then
+    echo 'Error: models or set of cmvn, lang_model, recog_model and decode_config are not set.' >&2
+    exit 1
+    fi
+fi
+
+dir=${download_dir}/${models}
+mkdir -p ${dir}
+
 function download_models () {
+    if [ -z $models ]; then
+        return 0
```

Comment:
Return value may be omitted.
```suggestion
        return
```